examples/2Dturbulence_multigrid/esel.cxx

```diff
@@ -14,7 +14,7 @@ class ESEL : public PhysicsModel {
   Field2D sigma_n, sigma_T, sigma_vort; // dissipation terms
   BoutReal zeta;                        // rho/R0
   BRACKET_METHOD bm;                    // Bracket method for advection terms
-  Laplacian* phiSolver;                 // Laplacian solver for vort -> phi
+  std::unique_ptr<Laplacian> phiSolver{nullptr};                 // Laplacian solver for vort -> phi
```

Comment:
Is there any value in trying to define a type here? e.g define `Laplacian::pointer` to be `std::unique_ptr<Laplacian>`?
It hides the fact that it's a unique pointer, which is probably bad, but allows us to change the pointer type later if needed.

Reply:
We have done this before for a couple of types, but I've since been convinced it's probably less helpful than one might think. There's a big difference between getting a `unique_ptr`, a `shared_ptr` and a raw pointer, and we probably shouldn't hide that.